docs/architecture/adr-028-libp2p.md

```diff
@@ -19,7 +19,7 @@
 
 > A decision may be "proposed" if it hasn't been agreed upon yet, or "accepted" once it is agreed upon. If a later ADR changes or reverses a decision, it may be marked as "deprecated" or "superseded" with a reference to its replacement.
```

Comment:
```suggestion
```

Reply:
i really liked the current ADR though 😉.

Ill clean up the empty adrs